docs/csharp/discards.md

```diff
@@ -36,9 +36,9 @@ For more information on deconstructing user-defined types with discards, see [De
 
 ## Pattern matching with `switch`
 
-The *discard pattern* can be used in pattern matching with the [switch](language-reference/keywords/switch.md) keyword. Every expression always matches the discard pattern. (It can be used with [is](language-reference/keywords/is.md) expressions. However, that use is rare because the discard can be removed without changing its meaning).
```

Comment:
> (It can be used with [is](language-reference/keywords/is.md) expressions. However, that use is rare because the discard can be removed without changing its meaning).

This removed sentence seems correct (but probably not clear/confusing). I believe it's referring to `if (obj is SomeType _)`.

Reply:
It's not the discard pattern. It's the declaration pattern with a discard.